src/infcholesky.jl

```diff
@@ -18,6 +18,8 @@ end
 AdaptiveCholeskyFactors(A::AbstractMatrix{T}) where T = AdaptiveCholeskyFactors(MemoryLayout(A), A)
 MemoryLayout(::Type{AdaptiveCholeskyFactors{T,DM,M}}) where {T,DM,M} = AdaptiveLayout{typeof(MemoryLayout(DM))}()
 
+copy(A::AdaptiveCholeskyFactors) = A
```

Comment:
`partialcholesky!()` was keeping track of an `ncols` that could not be updated if it wasn't mutable. But since data is a cached array I don't remember why this is needed and not redundant, I will see if it can be removed

Reply:
Oh I see. It's fine just leave it.